packages/commerce-sdk-react/package-lock.json

```diff
@@ -5,9 +5,9 @@
 	"requires": true,
 	"dependencies": {
 		"@adobe/css-tools": {
-			"version": "4.0.2",
```

Comment:
No idea. Could be because of slight npm versions changes, could be because of caching. Hard to really say to be honest. 
I can put it back if you like, but I imaging that the lock files will be re-generate a few times before this branch is merged into develop